test/storybook-playwright/storybook/main.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+const baseConfig = require( '../../../storybook/main' );
+
+const config = {
+	...baseConfig,
+	addons: [ '@storybook/addon-toolbars' ],
+	stories: [
+		'../../../packages/components/src/**/e2e.stories.@(js|tsx|mdx)',
```

Comment:
Do we need to take any extra precautions in excluding the `e2e` stories from being included in the normal Storybook build?

Reply:
Whoops, fixed 👍 

Changed the pattern to be `stories/e2e/*.js` instead of `e2e.stories.js` to make this simpler.